package.json

```diff
@@ -22,19 +22,23 @@
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-flow-comments": "^6.17.0",
     "babel-plugin-transform-object-rest-spread": "^6.16.0",
+    "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.18.0",
+    "babel-preset-es2016": "^6.16.0",
+    "babel-preset-es2017": "^6.16.0",
     "babel-register": "^6.18.0",
     "chai": "^3.5.0",
-    "eslint": "^3.9.1",
-    "eslint-config-canonical": "^5.4.0",
-    "husky": "^0.11.9"
+    "eslint": "^3.10.2",
+    "eslint-config-canonical": "^5.5.0",
+    "husky": "^0.11.9",
+    "npm-watch": "^0.1.6"
   },
   "keywords": [
     "youtube",
     "iframe",
     "player",
     "api",
-    "promise",
```

Comment:
Whats the reason for removing "promise" keyword?

async/ await still returns a promise.